### ISSUE TITLE
how should it be used in eval mode?

### ISSUE THREAD

**Issue by greaber** [dannysdeng/selu#3]
With standard dropout, you use all the units during evaluation and scale the output to match what you used in training.  The paper doesn't explicitly say how you should treat alpha dropout during evaluation.  Given that it is designed to preserve the statistics of the input, I would guess that maybe you can just skip the alpha dropout layers (or set p=0) during evaluation, but is this definitely correct?  Should the layer do this automatically when the model is in eval mode?

**Comment by dannysdeng** [comment]
Thanks for the comment. As in the TensorFlow implementation, the lines [33-35](https://github.com/dannysdeng/selu/blob/master/selu.py#L33) should have addressed the case during evaluation. 

For further information, you may proceed to check out the PyTorch official implementation [here](http://pytorch.org/docs/master/nn.html#alphadropout).
